oak_runtime/src/node/http/server.rs

```diff
@@ -358,6 +362,21 @@ impl HttpRequestHandler {
     }
 }
 
+/// Check if the request contains a `TRANSFER_ENCODING` header, and reject the request in that case
+/// by returning an error.
+fn validate_request(req: &HttpRequest) -> anyhow::Result<()> {
```

Comment:
@conradgrobler Is this the right way to reject the requests with a TRANSFER_ENCODING header?

Reply:
It looks right, apart from seeming a bit more complex than needed as @tiziano88 suggested below.